### PR TITLE
Extend roadmap: multi page tests

### PR DESCRIPTION
**Pull request by OrKoN**
Documenting the use case for the spec: https://w3c.github.io/webdriver-bidi/#command-browsingContext-activate

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -90,3 +90,10 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
+
+### Multi page automation
+
+This scenario creates multiple web pages at the same time and performs tests that require switching between multple pages.
```
Comment (mathiasbynens) [inline roadmap.md:96]:
```suggestion
This scenario opens multiple web pages at the same time, and performs tests that require switching between multiple pages.
```

**Review by mathiasbynens — APPROVED** [review]

**Review by thiagowfx — APPROVED** [review]

File: roadmap.md [hunk 2]
```diff
@@ -90,3 +90,10 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
+
+### Multi page automation
+
+This scenario opens multiple web pages at the same time, and performs tests that require switching between multiple pages.
```
[the remark this answers is not in the document]
Comment (OrKoN) [reply roadmap.md:96]: > As it sounds now the tab to interact with always needs to be activated. But that is actually not the case. Tests for multiple pages in parallel should also work fine while the page is in the background. So we probably should lift this restriction a bit?

How would you rephrase it?  Here, I only mention that the user performs tests that require switching between multiple pages. This does not make it impossible to test multiple page tests that don't require switching between pages (if that is what the user wants to test).

[the remark this answers is not in the document]
Comment (OrKoN) [reply roadmap.md:96]: > Would be good to see that overlapping navigations work fine and that the appropriate events are fired correctly. But maybe we can assume at this level that this works - but note that we do not actually have tests for that yet.

do we want to file a bug for this? From my perspective, the coverage is currently sufficient. Note that this only puts the use case on the roadmap. I think we all agree that it is something that should be supported by BiDi.

[the remark this answers is not in the document]
Comment (OrKoN) [reply roadmap.md:96]: @whimboo PTAL

File: roadmap.md [hunk 3]
```diff
@@ -91,6 +91,14 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
 
+### Multi page automation
+
+This scenario opens multiple web pages at the same time, and performs tests that require switching between multiple pages.
```
Comment (whimboo) [inline roadmap.md:96]:
```suggestion
This scenario opens multiple web pages at the same time, and performs tests that require switching between those pages.
```

**Review by whimboo — APPROVED** [review]
Sorry for the delay. Looks like this is overdue with the activate command already landed.

**Pull request merged**